forge.config.ts

```
@@ -9,10 +10,34 @@ import { FuseV1Options, FuseVersion } from "@electron/fuses";
 const now = new Date();
 const buildDate = now.toISOString().slice(0, 10).replace(/-/g, "");
 
+const packageJSON = JSON.parse(fs.readFileSync('./package.json', 'utf8'));
+const { version: packageVersion } = packageJSON;
+
+function resolveTargetAndVersion(branch: string): { target: string; version?: string } {
+  if (branch === "main") {
+    return { target: "test" };
+  }
+
+  const releaseRcMatch = branch.match(/^release\/([\d]+\.\d+\.\d+-rc-\d+)$/);
```

ESLint

[error] Replace this character class by the character itself.

(sonarjs/single-char-in-character-classes)